packaging/rpm/flightctl.spec

```diff
@@ -35,7 +41,7 @@ GOENVFILE=$(go env GOROOT)/go.env
 if [[ ! -f "{$GOENVFILE}" ]]; then
     export GOPROXY='https://proxy.golang.org,direct'
 fi
-make build
+SOURCE_GIT_TAG=%{source_git_tag} SOURCE_GIT_COMMIT=%{source_git_commit} make build
```

Comment:
I suspect this won't work, because the .git dir is not included in the tar.gz shipped inside the srpm file.

A srpm file (high level) is a package that includes the spec and the tar.gz with all the source files. But git won't be able to get the tags I believe.

Reply:
yep, it doesn't :(